### PR TITLE
Issue #38 - Ensuring single + multi-measure cubes pass QB/SKOS SPARQL test constraints.

### PR DESCRIPTION
**Pull request by robons**

### CODE REVIEW

File: devtools/devtools/behave/temporarydirectory.py [hunk 1]
```diff
@@ -14,8 +18,20 @@ def get_context_temp_dir_path(context) -> Path:
 
         context.add_cleanup(lambda: context.temp_dir.cleanup())
 
-        context.add_cleanup(
-            lambda: shutil.move(context.temp_dir.name, "/root/last_run_behave")
-        )
+        if "KEEP_FILES" in os.environ and bool(distutils.util.strtobool(os.environ["KEEP_FILES"])):
```
Comment (robons) [inline devtools/devtools/behave/temporarydirectory.py:21]: @canwaf - I've hidden your functionality for keeping the files behind here and tidied it up a bit so you don't end up with all those `tmp` folders building up in there.

File: csvqb/csvqb/models/cube/qb/catalog.py [hunk 2]
```diff
@@ -32,7 +33,7 @@ def get_description(self) -> Optional[str]:
         return self.description
 
     def get_identifier(self) -> str:
-        return self.title
+        return self.identifier or self.title
```
Comment (santhosh-thangavel) [inline csvqb/csvqb/models/cube/qb/catalog.py:36]: why we need an identifier?

File: csvqb/csvqb/tests/behaviour/qbwriter.feature [hunk 3]
```diff
@@ -1,15 +1,15 @@
 Feature: Test outputting CSV-Ws with Qb flavouring.
 
   Scenario: A QbCube should generate appropriate DCAT Metadata
-    Given a single-measure QbCube named "Some Qube"
+    Given a single-measure QbCube with identifier "qb-id-10002" named "Some Qube"
```
Comment (santhosh-thangavel) [inline csvqb/csvqb/tests/behaviour/qbwriter.feature:4]: If an identifier is not given, would we have a QbCube with a title?

Comment (robons) [reply csvqb/csvqb/tests/behaviour/qbwriter.feature:4]: Yes, `title` is a required property. And if the user doesn't provide it, then they're ignoring all of our advice.

File: csvqb/csvqb/models/cube/qb/catalog.py (hunk 2 above)
Comment (robons) [reply csvqb/csvqb/models/cube/qb/catalog.py:36]: So the identifier just allows us to differentiate between datasets on the SPARQL end-point without having to use the title. And it gives the user an extra degree of flexibility to specify an identifier that's better/more specific than the title.

**Pull request merged**